tests/components/teksavvy/test_sensor.py

```diff
@@ -47,43 +52,43 @@ async def test_capped_setup(hass, aioclient_mock):
     await hass.async_block_till_done()
 
     state = hass.states.get("sensor.teksavvy_data_limit")
-    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == DATA_GIGABYTES
     assert state.state == "400"
 
     state = hass.states.get("sensor.teksavvy_off_peak_download")
-    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == DATA_GIGABYTES
     assert state.state == "36.24"
 
     state = hass.states.get("sensor.teksavvy_off_peak_upload")
-    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == DATA_GIGABYTES
     assert state.state == "1.58"
 
     state = hass.states.get("sensor.teksavvy_off_peak_total")
-    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == DATA_GIGABYTES
     assert state.state == "37.82"
 
     state = hass.states.get("sensor.teksavvy_on_peak_download")
-    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == DATA_GIGABYTES
     assert state.state == "226.75"
 
     state = hass.states.get("sensor.teksavvy_on_peak_upload")
-    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == DATA_GIGABYTES
     assert state.state == "8.82"
 
     state = hass.states.get("sensor.teksavvy_on_peak_total")
-    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == DATA_GIGABYTES
     assert state.state == "235.57"
 
     state = hass.states.get("sensor.teksavvy_usage_ratio")
-    assert state.attributes.get("unit_of_measurement") == PERCENTAGE
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == PERCENTAGE
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)